src/components/BasketballCourt/Border.tsx

```diff
@@ -9,18 +10,21 @@ const Border: React.FC<BorderProps> = ({ startPoint }) => {
   const { courtAreaXLength, courtAreaYLength, borderLength } = useStoreSelector(
     (state) => state.courtSize
   );
+  const currentColorBoardColor = useStoreSelector((state) => state.courtColor.color);
+  const [courtBorderColor, setCourtAreaColor] = useState("#195955");
```

Comment:
This naming convention may not good for hooks. Normally people use [something, useSomething] define useState. These 2 destructured names should be the same apart from "use".